### PR TITLE
Feature/items

### PR DESCRIPTION
**Pull request by AlanFletes**
Added all the ore chunk items and the correspondig loot tables for the drops. Also, the lang file has been updated with the name of the blocks and items added in past pulls.

### CODE REVIEW

**Comment by Caeleron** [comment]
Noticed a few ores were added that duplicate existing Minecraft ores. In particular, diamond, lignite and native gold are duplicates of Minecraft's diamond, coal and gold ore respectively. Native copper wasn't a duplicate, but since we ultimately will be targetting 1.17, which does have it, I see it as a duplicate ore.

**Comment by jrddunbr** [comment]
We should keep the copper ore for now. The rest of the duplicates can be removed.

File: integration-mc1-16/src/main/kotlin/org/eln2/ChunkItems.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import net.minecraft.item.Item
+import org.eln2.items.MultimeterItem
+import org.eln2.items.OreChunks
+
+/**
+ * Items added here are automatically registered.
+ */
+enum class ChunkItems(val items: Item) {
+    MULTIMETER(MultimeterItem()),
```
Comment (jrddunbr) [inline integration-mc1-16/src/main/kotlin/org/eln2/ChunkItems.kt]: This is not a chunk item and should be added in a new enum class called `MiscItems`.

File: integration-mc1-16/src/main/kotlin/org/eln2/ModBlocks.kt [hunk 2]
```diff
@@ -1,71 +1,4 @@
...
-    ORE_SMITHSONITE(OreBlock(OreData(1.0f,"ore_smithsonite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_SPHALERITE(OreBlock(OreData(1.0f,"ore_sphalerite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_SPHEROCOBALTITE(OreBlock(OreData(1.0f,"ore_spherocobaltite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_SPODUMENE(OreBlock(OreData(1.0f,"ore_spodumene",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_STANNITE(OreBlock(OreData(1.0f,"ore_stannite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_SYLVITE(OreBlock(OreData(1.0f,"ore_sylvite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_TANTALITE(OreBlock(OreData(1.0f,"ore_tantalite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_TITANITE(OreBlock(OreData(1.0f,"ore_titanite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance))),
-    ORE_WOLFRAMITE(OreBlock(OreData(1.0f,"ore_wolframite",1.0F,ModItems.ORE_CHUNKS_COPPER.items.defaultInstance)))
+class ModBlocks {
```
Comment (jrddunbr) [inline integration-mc1-16/src/main/kotlin/org/eln2/ModBlocks.kt]: This class can be deleted.

File: integration-mc1-16/src/main/kotlin/org/eln2/ModItems.kt [hunk 3]
```diff
@@ -1,13 +1,4 @@
...
-import org.eln2.items.MultimeterItem
-import org.eln2.items.OreChunks
-
-/**
- * Items added here are automatically registered.
- */
-enum class ModItems(val items: Item) {
-    MULTIMETER(MultimeterItem()),
-    ORE_CHUNKS_COPPER(OreChunks())
+class ModItems {
```
Comment (jrddunbr) [inline integration-mc1-16/src/main/kotlin/org/eln2/ModItems.kt]: This class can be deleted

File: integration-mc1-16/src/main/kotlin/org/eln2/utils/OreData.kt [hunk 4]
```diff
@@ -8,6 +8,12 @@ import net.minecraft.item.ItemStack
  *
  * @param hardness How hard is the block (pickaxe type basically)
  * @param name Name of the ore (eg, "native_copper")
+ * @param rarity Rarity of the ore (eg, ->
+ * Very Rare 0 - 24
+ * Rare 25 - 49
+ * Uncommon 50 - 73
+ * Common 74 - 97
+ * Very Common 98 - 120)
```
Comment (jrddunbr) [inline integration-mc1-16/src/main/kotlin/org/eln2/utils/OreData.kt]: This needs units, for example, ore veins per chunk.

**Review by jrddunbr — CHANGES_REQUESTED** [review]
This and the comments above about duplicates; remove all of the duplicates except for the native copper ore. We will remove that when we are actually on 1.17.

**Comment by jrddunbr** [comment]
I can't seem to find the registration location for the Flubber block. But this change is definitely a step in the right direction.

**Comment by jrddunbr** [comment]
I just did a force push with a complete rebase of all of the commits that were here. Going to start testing.

**Review by jrddunbr — APPROVED** [review]
LGTM. Fixed the item texture for Flubber and did some quick registry edits in another commit.

**Pull request merged**